src/core/controllers/bot.ts

```diff
@@ -72,7 +77,7 @@ bot.command('transactions', async (ctx) => {
   } else {
     await ctx.reply('Not an Admin. You do not have access to this command');
   }
-});
+});*/
```

Comment:
Why did you comment out this command?

Reply:
cause i have it ready for the button 
i will be deleting it in the next commit